modules/stars.py

```diff
@@ -20,8 +20,263 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
+import asyncpg
+import discord
+import datetime
+from discord.ext import commands
+
 import core
 
+CONFIG = core.CONFIG["STARBOARD"]
+STARBOARD_CHANNEL_ID = CONFIG.get("starboard_channel_id")
+
+STARBOARD_EMBED_COLOR = 0xFFFF00
+STARBOARD_EMOJI = "⭐"
+HEADER_TEMPLATE = "**{}** {} in: <#{}> ID: {}"
+
+VALID_FILE_ATTACHMENTS = (".jpg", ".jpeg", ".png", ".webp", ".gif")
+VIDEO_FILE_ATTACHMENTS = (".mp4", ".mov")
+VALID_IMAGE_LINKS = ("https://images-ext-1.discordapp.net", "https://tenor.com/view/")
+
+
+class JumpView(discord.ui.View):
+    def __init__(
+        self,
+        *,
+        timeout: float,
+        url: str | None,
+        label_name: str = "Jump to message",
+    ):
+        super().__init__(timeout=timeout)
+        self.add_item(discord.ui.Button(url=url, label=label_name, style=discord.ButtonStyle.primary))
+
+
+class StarboardEntry:
+    exists: bool = False
+    msg_id: int = 0
+    channel_id: int = 0
+    stars: int = 0
+    bot_message_id: int = 0
+    bot_content_id: int = 0
+
+    def __init__(self, db: asyncpg.Pool, msg_id: int):  # type: ignore
+        self.msg_id = msg_id
+        self.db: asyncpg.Pool[asyncpg.Record] = db
+
+    async def fetch(self):
```

Comment:
```suggestion
    async def fetch(self) -> None:
```

Return annotations are good.